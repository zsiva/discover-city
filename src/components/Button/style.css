.btn {
  background-color: #4caf50; /* Green */
  border: none;
  color: white;
  width: 100%;
  max-width: 200px;
}
.medium {
  padding: 15px 0;
}
.small {
  padding: 10px 20px;
}
