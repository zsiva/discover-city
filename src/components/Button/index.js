import React, { Component } from 'react';

import './style.css';

class Button extends Component {
  render() {
    const {handleClick, label} = this.props;

    return (
      <button
        className="btn"
        onClick={handleClick}>{label}</button>
    );
  }
}

export default Button;
