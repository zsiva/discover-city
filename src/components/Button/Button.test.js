import React from 'react';
import renderer from 'react-test-renderer';

import Button from './';

describe('Custom button', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<Button name="edit" />);
    expect(tree).toMatchSnapshot();
  });
});
