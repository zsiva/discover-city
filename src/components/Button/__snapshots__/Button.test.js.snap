// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Custom button should match snapshot 1`] = `
<button
  className="btn  medium"
  disabled={false}
  onClick={undefined}
  style={undefined}
  type="button"
/>
`;
